src/components/LowerLayout/GeneralInfo/GeneralInfo.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import global from '../../../styles.module.css';
+import classes from './GeneralInfo.module.css';
+
+const GeneralInfo = (props) => {
+  return (
+    <div className={global.container}>
+      {props.copy.details.map(copy => {
```

Comment:
This starts becoming a bit messy, I would pull these out into functions as follows:

Reply:
```const createList = listsCollection => listsCollection.map(({ list }) => (
  <ul key={list}>
    <li>{list}</li>
  </ul>
))


const createAdditionalInfo = additionalInfo => additionalInfo.map(({ info, link }) => (
  <p key={info}>
    {info}
    {link && <a href="https://search.opengazettes.org.za/text/25860?page=31">{link}</a>}
  </p>
))


const createDetails = (details, classes, listsContainer) => details.map(({ heading, firstParagraph }) => (
  <div className={classes.vessel} key={heading}>
    <h3 className={classes.heading}>{heading}</h3>
    <div className={classes.content}>
      <p>{firstParagraph}</p>
      {createList(listsContainer)}
      {additionalInfo && createAdditionalInfo(additionalInfo)}
    </div>
  </div>
))


const GeneralInfo = ({ details, lists: listsContainer, additionalInfo }) => (
  <div className={global.container}>
    {createDetails(details, classes, listsContainer, additionalInfo )}
  </div>
);


export default GeneralInfo;```

Reply:
Otherwise, it becomes easy to get variable names confused or get confused by open and closing JSX/HTML tags as well.